src/adv.py

```diff
@@ -49,3 +53,92 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
```

Comment:
Stefano , Great job on the repo and completing MVP . Your code looks great and I hope to see some stretch !  Keep up the great work!